src/components/dataset/overview/DatasetOverviewPanel.jsx

```diff
@@ -89,6 +89,13 @@ function DatasetOverviewPanel(props) {
             {...a11yProps(2)}
           />
         )}
+        <Tab
+          label="Roles & memberships"
```

Comment:
The ticket said to name this tab "access permissions", but the mock up said to name it "roles and memberships" - which one should we follow?

Reply:
@lstrano-broad, I'm guessing what's in mocks but I'd love to confirm